projects/timer_challenge/.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>timer_challenge</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+	</natures>
+</projectDescription>
```

Comment:
I think this was committed accidentally, what IDE are you using? We can add it to the .gitignore so it doesn't get committed in the future.